src/RemoteTech/UI/DebugWindow.cs

```diff
@@ -468,14 +468,24 @@ private void drawGuidReader()
 
             // draw the Ground stations
             #region Ground stations
-            foreach (var stations in RTCore.Instance.Network.GroundStations)
+            if (HighLogic.LoadedScene == GameScenes.SPACECENTER)
             {
                 GUILayout.BeginHorizontal();
+                GUILayout.Label("Ground stations are only available in the flight or tracking station.", GUILayout.ExpandWidth(true));
```

Comment:
Would you mind to change `in the flight or tracking station.`to `in the flight scene or tracking station.`?
Sorry for nitpicking :(